kong-3.6.0-0.rockspec

```diff
@@ -67,6 +67,7 @@ build = {
 
     ["kong.conf_loader"] = "kong/conf_loader/init.lua",
     ["kong.conf_loader.constants"] = "kong/conf_loader/constants.lua",
+    ["kong.conf_loader.utils"] = "kong/conf_loader/utils.lua",
```

Comment:
i'm okay with the overall change, but the name could be improved. a module named `utils` is usually  becoming a kitchen sink. should we name it more self explantory, something like parse or parsing_utils?

Reply:
rename to `parse.lua`.